api/spec/controllers/comments.spec.js

```diff
@@ -121,4 +134,243 @@ describe('/comments', () => {
       expect(response.body.token).toBe(undefined);
```

Comment:
Maybe the posts should be being reset between each test? Could get messy otherwise